bot.py

```diff
@@ -114,7 +139,23 @@ async def unsubscription_command(update: Update, context: ContextTypes.DEFAULT_T
 
 
 async def unknown(update: Update, context: ContextTypes.DEFAULT_TYPE):
-    await context.bot.send_message(chat_id=update.effective_chat.id, text="comando sconosciuto")
+    text = update.message.text.lower()
+
+    if text == 'start dubai' or text == 'start doc':
+        await menu_command(update, context)
```

Comment:
il comando menu_command si aspetta che il comando scritto dall'utente sia formattato così: menu__nomeristorante_

In questo caso gli viene passato "start dubai" e non è in grado di parsarlo. Bisogna aggiustare il parsing di menu_command oppure modificare il context che viene passato al metodo